### PR TITLE
Re-enable Contribute button for v1.1 projects set to 100% reserved rate

### PR DESCRIPTION
**Pull request by johnnyd-eth**
_

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](../CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](../CONTRIBUTING.md#browser-support).

### CODE REVIEW

File: src/components/Dashboard/Pay/index.tsx [hunk 1]
```diff
@@ -67,16 +67,10 @@ export default function Pay() {
           </Button>
         </Tooltip>
       )
-    } else if (
-      fcMetadata.reservedRate === 200 ||
-      fcMetadata.payIsPaused ||
-      overridePayDisabled
-    ) {
+    } else if (fcMetadata.payIsPaused || overridePayDisabled) {
```
Comment (aeolianeth) [inline src/components/Dashboard/Pay/index.tsx:70]: **issue** I think this check needs to be based on the contract version. We still want it disabled for v1 iiuc.

Comment (peripheralist) [reply src/components/Dashboard/Pay/index.tsx:70]: yep this is true. the check should basically be `disabled = (reservedRate == 200 && terminal.version == 1.1)`

Comment (johnnyd-eth) [reply src/components/Dashboard/Pay/index.tsx:70]: Oh yep misread this one! Fixed up, thanks

File: src/components/Dashboard/Pay/index.tsx [hunk 2]
```diff
@@ -90,6 +88,7 @@ export default function Pay() {
         </Tooltip>
       )
     } else {
+      console.log('currentFC.configured.eq(0): ', currentFC.configured.eq(0))
```
Comment (peripheralist) [inline src/components/Dashboard/Pay/index.tsx:91]: help i am a log and i am trapped in this diff

**Comment by peripheralist** [comment]
perfect, thanks @johnnyd-eth! just one comment 😘

File: src/components/Dashboard/Pay/index.tsx (hunk 2 above)
Comment (johnnyd-eth) [reply src/components/Dashboard/Pay/index.tsx:91]: I got you dawg

**Review by peripheralist — APPROVED** [review]

**Pull request merged**